cms/apps/media/models.py

```diff
@@ -212,10 +318,103 @@ class Video(models.Model):
         null=True
     )
 
+    external_video = models.CharField(
```

Comment:
This should be a URLField - rules out a class of bad input wherein something not-URL-like is entered.